clouddriver-kubernetes/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/caching/view/provider/KubernetesManifestProvider.java

```diff
@@ -45,6 +52,7 @@
 public class KubernetesManifestProvider {
   private final KubernetesCacheUtils cacheUtils;
   private final KubernetesAccountResolver accountResolver;
+  private final ExecutorService executorService = Executors.newCachedThreadPool();
```

Comment:
If you do this, the threads will have names, which is helpful for debugging... they'll show up in stack traces, for example:
```
Executors.newCachedThreadPool(
    ThreadFactoryBuilder().setNameFormat(getClass().getSimpleName() + "-%d").build()
  );
```

(I forgot to mention that on your last PR, sorry.)

Reply:
Oh, right, I've seen you do that before.  I'll update both places to do this.